release/RELEASE_GUIDELINES.md

```diff
@@ -0,0 +1,31 @@
+# Release Guidelines for the Onfido JS SDK
+We follow semver for versioning. Given a version number MAJOR.MINOR.PATCH, increment the:
+
+- MAJOR version when you make incompatible API changes
+- MINOR version when you add functionality in a backwards-compatible manner
+- PATCH version when you make backwards-compatible bug fixes
+
+# Prerequisite
+In order to perform the release you need to:
+1. Download the Web SDK release configuration secret file and store it as `release/releaseSecret.js`
+  - Edit secret file where necessary
+2. Make sure your laptop is set up to be able to login to S3
```

Comment:
How can we make sure? Some pointers would be helpful especially for the person who doesn't have much experience with the release.

Reply:
👌